### PR TITLE
feat(report-issue): update unwelcome concept prompt, add info alerts

### PR DESCRIPTION
**Pull request by wescopeland**
This PR:
* Adds an optional `templateKind` query param to _messages/create.blade.php_. When given, an info alert will be displayed and the recipient & subject fields will be forced to their pre-defined values.
* Updates the prompt for the unwelcome concept message to address [some of this feedback](https://discord.com/channels/476211979464343552/1002688331827658774/1249042351171698719).
* Adds achievement IDs to all subject lines generated by the phone tree buttons - ideally this could be used to help the respective teams quickly identify which achievement is being discussed / be used for future bot-driven improvements.

Screenshot 2024-06-08 at 1 43 26 PM

### CODE REVIEW

File: resources/views/pages/achievement/[achievement]/report-issue.blade.php [hunk 1]
```diff
@@ -98,64 +98,88 @@
...
-        $subject = urlencode("Typo: $achievement->title ({$achievement->game->title})");
-        $message = urlencode("I'd like to report a spelling/grammar error in [ach=$achievement->id]:\n(Describe the issue here)");
+        $subject = urlencode("Typo: {$achievement->title} [{$achievement->id}] ({$achievement->game->title})");
+        $message = urlencode("I'd like to report a spelling/grammar error in [ach={$achievement->id}]:\n(Describe the issue here)");
     @endphp
-    <x-ticket.guide-link buttonText="Message WritingTeam"
-                         href="{{ route('message.create') }}?to=WritingTeam&subject={{ $subject }}&message={{ $message }}">
+    <x-ticket.guide-link
+        buttonText="Message WritingTeam"
+        href="{{ route('message.create') }}?to=WritingTeam&subject={{ $subject }}&message={{ $message }}&templateKind='writing-error'"
```
Comment (Jamiras) [inline resources/views/pages/achievement/[achievement]/report-issue.blade.php:140]: Are the quotes intentional on `templateKind`?

Also occurs on other links

File: resources/views/pages/achievement/[achievement]/report-issue.blade.php [hunk 2]
```diff
@@ -98,64 +98,88 @@
...
 
     @php
-        $subject = urlencode("Unwelcome Concept: $achievement->title ({$achievement->game->title})");
-        $message = urlencode("I'd like to report an unwelcome concept in [ach=$achievement->id]:\n(Describe the issue here)");
+        $subject = urlencode("Unwelcome Concept: {$achievement->title} [{$achievement->id}] ({$achievement->game->title})");
+        $message = urlencode(
+            "I'd like to report an unwelcome concept in [ach={$achievement->id}].\n\n" .
+
+            "- Which Unwelcome Concept:\n" .
+            "[insert which concept from the docs here]\n\n" .
```
Comment (Jamiras) [inline resources/views/pages/achievement/[achievement]/report-issue.blade.php:162]: Other templates use parentheses and Capitalized sentences:
> (Describe the issue here)

File: resources/views/pages/achievement/[achievement]/report-issue.blade.php (hunk 1 above)
Comment (wescopeland) [reply resources/views/pages/achievement/[achievement]/report-issue.blade.php]: Nope - this was an accident, but still worked all the same. Removed them in latest.

File: resources/views/pages/achievement/[achievement]/report-issue.blade.php (hunk 2 above)
Comment (wescopeland) [reply resources/views/pages/achievement/[achievement]/report-issue.blade.php]: Fixed in latest

File: resources/views/pages/achievement/[achievement]/report-issue.blade.php [hunk 3]
```diff
@@ -98,64 +98,88 @@
...
-        $message = urlencode("I'd like to report an unwelcome concept in [ach=$achievement->id]:\n(Describe the issue here)");
+        $subject = urlencode("Unwelcome Concept: {$achievement->title} [{$achievement->id}] ({$achievement->game->title})");
+        $message = urlencode(
+            "I'd like to report an unwelcome concept in [ach={$achievement->id}].\n\n" .
+
+            "- Which Unwelcome Concept:\n" .
+            "(Insert which concept from the docs here)\n\n" .
+
+            "- Detailed Explanation:\n" .
+            "(provide as much detail as possible here. Assume the reader may not have played the game before. The more detail you provide, the better your case.)"
```
Comment (Jamiras) [inline resources/views/pages/achievement/[achievement]/report-issue.blade.php:165]: First word "provide" is still uncapitalized.

Comment (wescopeland) [reply resources/views/pages/achievement/[achievement]/report-issue.blade.php]: Looks like both this one was uncapitalized as well as the one for the unwelcome concept prompt. Fixed in latest.

**Review by Jamiras — APPROVED** [review]

**Pull request merged**